verticapy/sdk/vertica/udf/gen.py

```diff
@@ -74,6 +88,23 @@ def generate_lib_udf(
     -------
     udx_str, sql
         UDF py file, str needed to install the library.
+
+    Example
+    -------
+    .. ipython:: python
+
+        from verticapy.udf import generate_lib_udf
+        @suppress
+        import math
+
+        udx_str, udx_sql = generate_lib_udf([(math.exp, [float], float, {}, "python_exp"),
```

Comment:
please use PEP8 formatting:
```
udx_str, udx_sql = generate_lib_udf(
    [(math.exp, [float], float, {}, "python_exp"), (math.isclose, [float, float], bool, {"abs_tol": float}, "python_isclose"),],
    library_name = "python_math",
    file_path = "",
    create_file = False
)
```
@mail4umar or @afard is it possible to use black on RSTs? Specifically for python cells.